Disable stdout/stderr redirect during RPM transactions

If this is enabled, it conflicts with the stdout/stderr used by anaconda.

This should only be used when Anaconda is being used, and not generally used.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

Rebase to 66b484fbc6190b4c99e5428ad26c0ce984e57eaf in oe-core
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 smart/backends/rpm/pm.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/smart/backends/rpm/pm.py b/smart/backends/rpm/pm.py
index 3286e5d..c066dd9 100644
--- a/smart/backends/rpm/pm.py
+++ b/smart/backends/rpm/pm.py
@@ -240,14 +240,14 @@ class RPMPackageManager(PackageManager):
             probfilter |= rpm.RPMPROB_FILTER_REPLACENEWFILES
         ts.setProbFilter(probfilter)
         cb = RPMCallback(prog, upgradednames)
-        cb.grabOutput(True)
+        #cb.grabOutput(True)
         probs = None
         retry = 0
         try:
             probs = ts.run(cb, None)
         finally:
             del getTS.ts
-            cb.grabOutput(False)
+            #cb.grabOutput(False)
             if (probs is not None) and sysconf.has("attempt-install", soft=True):
                 def remove_conflict(pkgNEVR):
                     for key in changeset.keys():
-- 
2.8.1

